### PR TITLE
[IOTDB-1077] add insertOneDeviceRecords API in java session

### PR DESCRIPTION
**Pull request by Ring-k**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java [hunk 1]
```diff
@@ -2503,6 +2504,52 @@ public TsFileManagement getTsFileManagement() {
     return tsFileManagement;
   }
 
+
+  public void insert(InsertRowsOfOneDevicePlan insertRowsOfOneDevicePlan)
+      throws WriteProcessException {
+//    for (InsertRowPlan plan : insertRowsOfOneDevicePlan.getRowPlans()) {
+//      insert(plan);
+//    }
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java:2512]: Remove these?

File: server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java [hunk 2]
```diff
@@ -361,7 +361,7 @@ public static PhysicalPlan create(ByteBuffer buffer) throws IOException, Illegal
     REVOKE_USER_PRIVILEGE, GRANT_ROLE_PRIVILEGE, GRANT_USER_PRIVILEGE, GRANT_USER_ROLE, MODIFY_PASSWORD, DELETE_USER,
     DELETE_STORAGE_GROUP, SHOW_TIMESERIES, DELETE_TIMESERIES, LOAD_CONFIGURATION, CREATE_MULTI_TIMESERIES,
     ALTER_TIMESERIES, FLUSH, CREATE_INDEX, DROP_INDEX,
-    CHANGE_TAG_OFFSET, CHANGE_ALIAS, MNODE, MEASUREMENT_MNODE, STORAGE_GROUP_MNODE
+    CHANGE_TAG_OFFSET, CHANGE_ALIAS, MNODE, MEASUREMENT_MNODE, STORAGE_GROUP_MNODE,BATCH_INSERT_ONE_DEVICE
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java:364]:
```suggestion
    CHANGE_TAG_OFFSET, CHANGE_ALIAS, MNODE, MEASUREMENT_MNODE, STORAGE_GROUP_MNODE, BATCH_INSERT_ONE_DEVICE
```

File: session/src/main/java/org/apache/iotdb/session/Session.java [hunk 3]
```diff
@@ -558,6 +561,112 @@ public void insertRecords(List<String> deviceIds, List<Long> times,
...
+    return Arrays.asList(result);
+  }
+
+  private List<ByteBuffer> objectValuesListToByteBufferList(List<List<Object>> valuesList, List<List<TSDataType>> typesList)
+      throws IoTDBConnectionException {
+    List<ByteBuffer> buffersList = new ArrayList<>();
+    for (int i = 0; i < valuesList.size(); i++) {
+      ByteBuffer buffer = ByteBuffer.allocate(calculateLength(typesList.get(i), valuesList.get(i)));
+      putValues(typesList.get(i), valuesList.get(i), buffer);
+//      buffer.flip();
```
Comment (SilverNarcissus) [inline session/src/main/java/org/apache/iotdb/session/Session.java:663]: remove this?

**Review by SilverNarcissus — CHANGES_REQUESTED** [review]
Fantastic work! Only one issue please have a look~

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java (hunk 1 above)
Comment (Ring-k) [reply server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java]: It's removed. Thank you for the suggestion!

File: session/src/main/java/org/apache/iotdb/session/Session.java (hunk 3 above)
Comment (Ring-k) [reply session/src/main/java/org/apache/iotdb/session/Session.java]: It's removed. Thank you for the suggestion!

File: server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java (hunk 2 above)
Comment (Ring-k) [reply server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java]: I changed it. Thank you for the suggestion!

**Pull request merged**